.github/workflows/acceptance-tests.yml

```diff
@@ -22,52 +22,54 @@ jobs:
     name: Analyze deploy string
     runs-on: ubuntu-24.04
     outputs:
-      pr-contains-string: ${{ steps.deploy-comment.outputs.pr-contains-string }}
-      renku: ${{ steps.deploy-comment.outputs.renku}}
-      renku-core: ${{ steps.deploy-comment.outputs.renku-core}}
-      renku-gateway: ${{ steps.deploy-comment.outputs.renku-gateway}}
-      renku-graph: ${{ steps.deploy-comment.outputs.renku-graph}}
-      renku-notebooks: ${{ steps.deploy-comment.outputs.renku-notebooks}}
-      renku-ui: ${{ steps.deploy-comment.outputs.renku-ui}}
-      amalthea-sessions: ${{ steps.deploy-comment.outputs.amalthea-sessions}}
-      amalthea: ${{ steps.deploy-comment.outputs.amalthea}}
-      test-enabled: ${{ steps.deploy-comment.outputs.test-enabled}}
-      test-legacy-enabled: ${{ steps.deploy-comment.outputs.test-legacy-enabled}}
-      extra-values: ${{ steps.deploy-comment.outputs.extra-values}}
+      renku: ${{ steps.deploy-comment.outputs.renku }}
+      deploy: ${{ steps.deploy-comment.outputs.pr-contains-string }}
+      renku-core: ${{ steps.deploy-comment.outputs.renku-core }}
+      renku-gateway: ${{ steps.deploy-comment.outputs.renku-gateway }}
+      renku-graph: ${{ steps.deploy-comment.outputs.renku-graph }}
+      renku-notebooks: ${{ steps.deploy-comment.outputs.renku-notebooks }}
+      renku-ui: ${{ steps.deploy-comment.outputs.renku-ui }}
+      renku-data-services: ${{ steps.deploy-comment.outputs.renku-data-services }}
+      amalthea: ${{ steps.deploy-comment.outputs.amalthea }}
+      amalthea-sessions: ${{ steps.deploy-comment.outputs.amalthea-sessions }}
+      test-enabled: ${{ steps.deploy-comment.outputs.test-enabled == 'true' }}
+      test-legacy-enabled: ${{ steps.deploy-comment.outputs.test-legacy-enabled == 'true' }}
+      extra-values: ${{ steps.deploy-comment.outputs.extra-values }}
     steps:
       - id: deploy-comment
-        uses: SwissDataScienceCenter/renku-actions/check-pr-description@v1.18.2
+        uses: SwissDataScienceCenter/renku-actions/check-pr-description@v1.19.0
         with:
           pr_ref: ${{ github.event.number }}
 
   deploy-pr:
-    name: Deploy
+    name: Deploy on Azure
     runs-on: ubuntu-24.04
     needs: [check-deploy]
     permissions:
       pull-requests: write
-    if: github.event.action != 'closed' && needs.check-deploy.outputs.pr-contains-string == 'true'
-    environment:
-      name: renku-ci-ds-${{ github.event.number }}
-      url: https://renku-ci-ds-${{ github.event.number }}.dev.renku.ch
```

Comment:
Can you put this back in? It's pretty useful to have on the PR.

Reply:
I had to revert that change, as it impacts the subject claim during the azure login process. I want to get it working as it used to yesterday, and then I'll validate if I can have these in the environment or not.

Reply:
Can you not match the environment with a wildcard? This should be possible. It would be extremely disappointing to me that you all can't figure out how to make GitHub environments work, as they provide valuable features for dev.

Reply:
The first § states exactly the problem we had a couple of weeks back (at least from memory what I heard being discussed) and at the time there was no solution for that problem. I don't know what it implies to try and implement this right now in terms of effort.

Reply:
I think this is the only unersolved problem here. And it seems that the environment part does not work. I saw that in the ui you commented that out @leafty . I think that means this can be approved and merged?